src/commands/profiles/list.js

```diff
@@ -4,16 +4,26 @@ const { BaseCommand } = require('@twilio/cli-core').baseCommands;
 class ProfilesList extends BaseCommand {
   async run() {
     await super.run();
+    const envProfile = this.userConfig.getProfileFromEnvironment();
+    // If environment profile exists, add required details to userConfig.profiles, and mark as active.
+    if (envProfile) {
+      const { accountSid: ENVIRONMENT_ACCOUNT_SID, region: ENVIRONMENT_REGION } = envProfile;
+      const strippedEnvProfile = {
+        id: '[env]',
+        accountSid: ENVIRONMENT_ACCOUNT_SID,
+        region: ENVIRONMENT_REGION,
+      };
+      this.userConfig.profiles.push(strippedEnvProfile);
+      this.userConfig.setActiveProfile(strippedEnvProfile.id);
```

Comment:
As we discussed, setting an active profile with the env vars will be an issue once the user removes the environment variable, it will still linger here.